apps/busybox/cvr_scaleup/test.yml

```diff
@@ -0,0 +1,247 @@
+              replace:
+                path: "./cvr.yml"
+                regexp: "<csp-uid>"
+                replace: "{{ uid.stdout }}"
+              
+            - name: Replacing the volume name in cvr yml
+              replace:
+                path: "./cvr.yml"
+                regexp: "<cstor-volume-name>"
+                replace: "{{ volume_name.stdout }}"
+
+            - name: Replacing the volume capacity in cvr yml
+              replace:
+                path: "./cvr.yml"
+                regexp: "<initial_capacity>"
+                replace: "{{ lookup('env','PV_CAPACITY') }}"
+
+            - name: Replacing the target IP in cvr yml
+              replace:
+                path: "./cvr.yml"
+                regexp: "<target-ip>"
+                replace: "{{ target_ip.stdout }}"
+
+            - name: Replacing the DRF in patch_cstorvolume yml
+              replace:
+                path: "./patch_cstorvolume.yml"
+                regexp: "<desired-replication-factor>"
+                replace: "3"
+
+            - name: Patching the cstorvolume's desired rep count
+              shell: kubectl patch cstorvolume {{ volume_name.stdout }} -n {{ operator_ns }} --type merge --patch "$(cat patch_cstorvolume.yml)"
+              register: patch_cstorvolume_result
+              failed_when: "'patched' not in patch_cstorvolume_result.stdout"
+            
+            - name: Creating new cvr
+              shell: kubectl apply -f cvr.yml
+
+            - name: Getting the uid of newly created cvr
+              shell: kubectl get cvr -n {{ operator_ns }} -l cstorpool.openebs.io/name={{ csp_name.stdout_lines[0] }},cstorvolume.openebs.io/name={{ volume_name.stdout }} -o jsonpath='{.items[0].metadata.uid}'
+              register: new_uid
+
+            - name: Creating MD5SUM of new-uid
+              shell: echo {{ new_uid.stdout }} | md5sum | awk '{print toupper($1)}' 
+              register: md5sum
+
+            - name: Replacing the md5sum value in patch yml
+              replace:
+                path: "./patch_cvr.yml"
+                regexp: "<md5-sum-uid>"
+                replace: "{{ md5sum.stdout }}"
+
+            - name: Patching the cvr's replica id with md5sum
+              shell: kubectl patch cvr {{ volume_name.stdout }}-{{ csp_name.stdout_lines[0] }} -n {{ operator_ns }} --type merge --patch "$(cat patch_cvr.yml)"
+              register: patch_cvr_result
+              failed_when: "'patched' not in patch_cvr_result.stdout"
+            
+            - name: Getting the cstor pool pod name
+              shell: kubectl get pod -n {{ operator_ns }} -l app=cstor-pool,openebs.io/cstor-pool={{ csp_name.stdout_lines[0] }} -o jsonpath='{.items[0].metadata.name}'
+              register: pool_pod_name
+
+            - name: Restarting the pool pod
+              shell: kubectl delete pod {{ pool_pod_name.stdout }} -n {{ operator_ns }}
+
+            - name: Checking for the pool pod to be in running state
```

Comment:
So if we honor above comment we no need to wait for this pod to come to running state

Reply:
@mittachaitu Sometimes this pod goes into completed state after killing the cstor-pool-mgmt. This is reason for this check.